frontend/src/app/graphql/hooks/useLegislationSearchQuery.ts

```diff
@@ -73,3 +141,59 @@ export const convertLegislationToOption = (legislation: Legislation[] | undefine
     })) ?? []
   );
 };
+
+/**
+ * Converts legislation items to hierarchical options for section dropdowns.
+ * Parts/Divisions are disabled group headers; Sections show "citation title".
+ */
+export const convertLegislationToHierarchicalOptions = (
```

Comment:
I thought bringing in the display order contextually by node hierarchy made the most sense (eg, all PARTS are ordered sequentially, but the children of PART 1 start at 1 again), but it does require somewhat convoluted logic to sort as one would expect.